src/frontend/src/lib/utils/wallet.utils.ts

```diff
@@ -45,18 +45,18 @@ export const cleanWorkers = ({
  * Load the worker for the token if it is not already loaded.
  *
  * @param workers The map of workers defined as `Map<TokenId, WalletWorker>`.
- * @param token The IC token to load the worker for.
+ * @param token The token to load the worker for.
  */
 export const loadWorker = async ({
 	workers,
 	token
 }: {
 	workers: Map<TokenId, WalletWorker>;
-	token: IcToken;
+	token: Token;
 }) => {
 	if (!workers.has(token.id)) {
 		const worker = await (token.standard === 'icrc'
-			? initIcrcWalletWorker(token)
+			? initIcrcWalletWorker(token as IcToken)
```

Comment:
This should remains in `$icp` in my opinion as it is solely related to the IC network. I mean the all function `loadWorker` or just those two `initIc...Worker` calls.

`wallet.utils.ts` should ideally not have references on `$icp`.

Reply:
Updated the function to hide the ICRC/ICP wallet workers initialisation logic. Though, `wallet.utils` still has one reference on `$icp` (`initWalletWorker`) and will have one on `$btc`, as my idea is to re-use `loadWorker` for BTC tokens, too. The next step would be to update the `IcLoaderWallet` component: it will be renamed to `LoaderWallet`, moved to `$lib/components/core` and is gonna accept tokens as a prop.

Reply:
We should pass the function to init as a parameter, that way it won't have any references.

Reply:
Sounds good, done.